subprojects/docs/src/samples/build-organization/standalone-precompiled-plugins/kotlin/src/main/kotlin/com.example.service.gradle.kts

```
@@ -0,0 +1,30 @@
+// tag::plugins[]
+plugins {
+    id("com.example.java-convention")
+}
+// end::plugins[]
+
+val integrationTest by the<SourceSetContainer>().creating {
+    compileClasspath += sourceSets["main"].output + sourceSets["test"].output
+    runtimeClasspath += sourceSets["main"].output + sourceSets["test"].output
+}
+
+val integrationTestTask = tasks.register<Test>("integrationTest") {
+    shouldRunAfter(tasks.named("test"))
+
+    testClassesDirs = sourceSets["integrationTest"].output.classesDirs
+    classpath = sourceSets["integrationTest"].runtimeClasspath
```

:x: the `integrationTest` source set is already in scope, prefer using the val instead of string references
```suggestion
    testClassesDirs = integrationTest.output.classesDirs
    classpath = integrationTest.runtimeClasspath
```